### PR TITLE
Submit score

### PR DESCRIPTION
**Pull request by audukehinde**
- Implemented Refresh button
- Implemented Submit button

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,125 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (muneebulrehman) [inline README.md]: - [ ] Kindly add a `Future Features` section, including 1-3 potential features to be added.
- [ ] The link to the MIT license is broken. Kindly fix it.

File: src/index.js [hunk 2]
```diff
@@ -1,2 +1,40 @@
...
+refreshScore.addEventListener('click', async () => {
+  const res = await fetch(gameUrl);
+
+  const data = await res.json();
+  let tableData = '';
+  data.result.forEach((values) => {
+    tableData += `<li> ${values.user}: ${values.score}</li>`;
+  });
+  document.getElementById('table-content').innerHTML = tableData;
+});
```
Comment (muneebulrehman) [inline src/index.js]: When loading the application the first time or after reload, the data should be fetched automatically and populated into the UI. However, in your case, it doesn't. The data is only populated when the user hits the refresh button. The refresh button is to be after submitting a new score and to display that score on UI. Kindly fix this issue.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @audukehinde , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Good UI.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — APPROVED** [review]
Hi @audukehinde 👋, 

BigTvGIF (2)


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅

- [ ] Linters are green👍
- [ ] Commit messages are meaningful👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**